File: app/src/main/java/com/coolweather/coolweather/gson/Suggestion.java
package com.coolweather.coolweather.gson;

import com.google.gson.annotations.SerializedName;

public class Suggestion {
    @SerializedName("comf")
    public Comfort comfort;
    public class Comfort{
        @SerializedName("txt")
        public String info;
    }

    @SerializedName("cw")
    public Carwash carwash;
    public class Carwash{
        @SerializedName("txt")
        public String info;
    }
    public Sport sport;
    public class Sport{
        @SerializedName("txt")
                public String info;
    }
}

File: app/src/main/java/com/coolweather/coolweather/gson/Suggestion.kt
package com.coolweather.coolweather.gson

import com.google.gson.annotations.SerializedName

public class Suggestion {
    @SerializedName("conf")
    var comfort=Comfort()
    inner class Comfort{
        @SerializedName("txt")
        var info=String
    }

    @SerializedName("cw")
    var carwash=Carwash()
    inner class Carwash{
        @SerializedName("txt")
        var info=String

    }
    var sport=Sport()
    inner class Sport{
        @SerializedName("txt")
        var info=String
    }
}